src/transformers/models/tapas/modeling_tapas.py

```diff
@@ -46,11 +46,17 @@
 from .configuration_tapas import TapasConfig
 
 
+logger = logging.get_logger(__name__)
+
 # soft dependency
 if is_scatter_available():
-    from torch_scatter import scatter
-
-logger = logging.get_logger(__name__)
+    try:
+        from torch_scatter import scatter
+    except OSError:
+        logger.info(
+            "`torch_scatter` can't be loaded. It seems you have a `torch_scatter` installed with the wrong CUDA version."
+            "Please try to reinstall it following the instructions here: https://github.com/rusty1s/pytorch_scatter"
+        )
```

Comment:
At this state the model won't be usable, so we should use the `error` level and specifically say that the TAPAS model is not usable.

This code should not be executed anymore if there is no `torch_scatter` installed and the user is not trying to load a TAPAS model.